pyspark/bigdl/nn/layer.py

```diff
@@ -115,14 +115,35 @@ def __call__(self, x=None):
                                      self,
                                      to_list(x)))
 
+    @staticmethod
+    def from_jvalue(jvalue, bigdl_type="float"):
+        """
+        Create a Python Model base on the given java value
+        :param jvalue: Java object create by Py4j
+        :return: A Python Model
+        """
+        model = Layer(jvalue=jvalue, bigdl_type=bigdl_type)
+        model.value = jvalue
+        return model
+
     @classmethod
     def of(cls, jvalue, bigdl_type="float"):
         """
-        Create a Python Layer base on the given java value
+        Create a Python Layer base on the given java value and the real type.
         :param jvalue: Java object create by Py4j
         :return: A Python Layer
         """
-        model = Layer(jvalue, bigdl_type)
+        realClassName = callBigDlFunc(bigdl_type,
+                      "getRealClassNameOfJValue",
+                      jvalue)
+        if realClassName == "StaticGraph" or realClassName == "DynamicGraph":
+            realClassName = "Model"
+        elif realClassName == "Input":
+            realClassName = "Layer"
```

Comment:
What if the real class is a `KerasModel`? Or in other words, if `Sequential`, how do you identify whether it is a Torch Sequential or a Keras Sequential?